app/src/sharedTest/java/org/oppia/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -64,6 +71,67 @@ class RevisionCardFragmentTest {
     }
   }
 
+  @Test
+  fun testReviewCardTestActivity_fractionSubtopicId1_checkReturnToTopicButtonIsDisplayedSuccessfully() {
+    launch<RevisionCardActivity>(
+      createReviewCardActivityIntent(
+        ApplicationProvider.getApplicationContext(),
+        FRACTIONS_TOPIC_ID,
+        SUBTOPIC_TOPIC_ID
+      )
+    ).use {
+      onView(withId(R.id.revision_card_return_button)).check(matches(withText("Return to topic")))
+    }
+  }
+
+  @Test
+  fun testReviewCardTestActivity_configurationChange_toolbarTitle_fractionSubtopicId1_isDisplayedCorrectly() {
+    launch<RevisionCardActivity>(
+      createReviewCardActivityIntent(
+        ApplicationProvider.getApplicationContext(),
+        FRACTIONS_TOPIC_ID,
+        SUBTOPIC_TOPIC_ID
+      )
+    ).use {
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
+      onView(
+        allOf(
+          instanceOf(TextView::class.java),
+          withParent(withId(R.id.revision_card_toolbar))
+        )
+      ).check(matches(withText("What is Fraction?")))
+    }
+  }
+
+  @Test
+  fun testReviewCardTestActivity_configurationChange_toolbarTitle_fractionSubtopicId1_checkExplanationAreDisplayedSuccessfully() {
```

Comment:
This does not match with the steps in your code.

Reply:
Done